src/twisted/test/proto_helpers.py

```diff
@@ -925,3 +927,64 @@ def _check():
     lc = task.LoopingCall(_check)
     lc.clock = reactor
     return lc.start(0.01, now=True)
+
+
+
+@implementer(ILogObserver)
+class EventLoggingObserver(Sequence):
+    """
+    L{ILogObserver} That stores its events in a list for later inspection.
+    This class is similar to L{LimitedHistoryLogObserver} save that the
+    internal buffer is public and intended for external inspection.  The
+    observer implements the sequence protocol to ease iteration of the events.
+
+    @ivar _events: The events captured by this observer
+    @type _events: L{list}
+    """
+    def __init__(self):
+        self._events = []
+
+
+    def __len__(self):
+        return len(self._events)
+
+
+    def __getitem__(self, index):
+        return self._events[index]
+
+
+    def __contains__(self, value):
```

Comment:
This __contains__ function is not getting covered.  Can you add unit test which exercises this function?

Reply:
Removing it would be easier, go ahead